modules/ztp-image-based-upgrade-with-backup.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrading-with-backup_{context}"]
+= Upgrading the {sno} cluster with {lcao}
+
+Once you generated the seed image and completed the `Prep` stage, you can upgrade the target cluster.
+During the upgrade process, the OADP Operator can create a backup of the artifacts if you specify them in the OADP CRs, then the {lcao} upgrades the cluster.
+
+[IMPORTANT]
+====
+If the upgrade fails or stops, an automatic rollback is initiated.
+If you have an issue after the upgrade, you can initiate a manual rollback.
+For more information about manual rollback, see "(Optional) Initiating rollback of the {sno} clusters after an image-based upgrade".
+====
+
+.Prerequisites
+
+* You generated a seed image for the target cluster.
+* You completed the `Prep` stage.
+
+.Procedure
+
+. Move to the upgrade stage by changing the value of the `stage` field to `Upgrade` in the `ImageBasedUpgrade` CR.
++
+[source,terminal]
+----
+$ oc patch imagebasedupgrades.lca.openshift.io upgrade -p='{"spec": {"stage": "Upgrade"}}' --type=merge
+----
+
+. Check the status of the `ImageBasedUpgrade` CR:
++
+[source,terminal]
+----
+$ oc get ibu -A -oyaml
+----
+
++
+.Example output
+[source,yaml]
+----
+status:
+  conditions:
+  - lastTransitionTime: "2024-01-01T09:00:00Z"
+    message: In progress
+    observedGeneration: 2
+    reason: InProgress
+    status: "False"
+    type: Idle
+  - lastTransitionTime: "2024-01-01T09:00:00Z"
+    message: 'Prep completed: total: 121 (pulled: 1, skipped: 120, failed: 0)'
+    observedGeneration: 2
+    reason: Completed
+    status: "True"
+    type: PrepCompleted
+  - lastTransitionTime: "2024-01-01T09:00:00Z"
+    message: Prep completed
+    observedGeneration: 2
+    reason: Completed
+    status: "False"
+    type: PrepInProgress
+  - lastTransitionTime: "2024-01-01T09:00:00Z"
+    message: Upgrade completed
+    observedGeneration: 3
+    reason: Completed
+    status: "True"
+    type: UpgradeCompleted
+----
+
+. The OADP Operator creates a backup of the data specified in the OADP `Backup` and `Restore` CRs.
+
+. The target cluster reboots.
+
+. Monitor the status of the CR:
++
+[source,terminal]
+----
+$ oc get ibu -A -oyaml
+----
+
+. The cluster reboots.
++
+--
+After the cluster reboots, the {lcao} does the following:
```

Comment:
from the new image installed during the prep stage

Reply:
Isn't the image only pulled during the prep stage? My understanding was that the installation happens during the upgrade stage. I can also just say:
> After the cluster reboots from the new image, the {lcao} does the following: